src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleControl.cs

```diff
@@ -2505,15 +2507,17 @@ internal static string GetConsoleWindowTitle()
         /// </exception>
         internal static void SetConsoleWindowTitle(string consoleTitle)
         {
+            if (!canSetConsoleWindowTitle)
```

Comment:
I'd revert  this variable to avoid static initialization.

```c#
internal static bool dontSetConsoleWindowTitle;

if (dontSetConsoleWindowTitle)

Reply:
In your case, `dontSetConsoleWindowTitle` would still be initialized to default value, so not clear what the difference will be here

Reply:
No, initialization with true add cctor but there is no cctor for default value
https://sharplab.io/#v2:C4LglgNgPgAgTARgLACgYGYAE9MGFMDeqmJmYAdsAKYBO5AhhNggGyYBGA9p0wCaeUAylWC4BAZx5UA6hX4B3ACphgEKpgC8mYDQCuVANzFSGbABZMAWQAUASkLGSAX1ROgA